### PR TITLE
Zeng Hu Augments: 'I Never Asked for This' Edition

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds some Zeng Hu exclusive augments.

Memory Inhibitor: A Zeng Hu implant that allows one to have control over their memories, allowing you to set a timer and remove any memories developed within it. A RP tool, no mechanical use.

Emotional Manipulator: A Zeng Hu brain implant to manipulate the brain’s chemicals to induce a calming or happy feeling. A RP tool, no mechanical use.

 Vision Enhanced Retinas: Allows the user to zoom at short distances as long they do not move. Always emits lights.

Ocular Installed Sightlights: Designed to assist medical personnel in darker areas or places experiencing periodic power issues, Sightlights will allow one to be able to use their eyes as a flashlight. 

 Zeng-Hu Veterancy Plate:  A clear sign of Zeng-Hu’s best, this plate bearing the company’s symbol is installed on those who prove themselves in the hyper-competitive environment.  Does nothing but look cool.

Also, tweaks augment prices a bit. Augments that offer no advantage are now only one point.

### CODE REVIEW

File: code/__defines/mobs.dm [hunk 1]
```diff
@@ -157,6 +157,10 @@
 #define BP_AUG_LANGUAGE     "integrated language processor"
 #define BP_AUG_PSI         "psionic receiver"
 #define BP_AUG_CALF_OVERRIDE     "calf overdrive"
+#define BP_AUG_MEMORY         "memory inhibitor"
+#define BP_AUG_EMOTION         "emotional manipulator"
+#define BP_AUG_ENCHANED_VISION   "vision enhanced retinas"
+#defineBP_AUG_SIGHTLIGHTS   "ocular installed sightlights"
```
Comment (1Sparky1) [inline code/__defines/mobs.dm:163]:
```suggestion
#define BP_AUG_SIGHTLIGHTS   "ocular installed sightlights"
```
Missing a space here.

File: code/modules/client/preference_setup/loadout/loadout_augments.dm [hunk 2]
```diff
@@ -66,11 +62,47 @@
...
 
 /datum/gear/augment/psiaug
 	display_name = "psionic receiver"
 	description = "An augment installed into the head that functions as a surrogate for a missing zona bovinae, also functioning as a filter for the psionically-challenged."
 	path = /obj/item/organ/internal/augment/psi
-	cost = 1
 	whitelisted = list(SPECIES_HUMAN, SPECIES_HUMAN_OFFWORLD, SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_WARRIOR, SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+
+/datum/gear/augment/memory_inhibitor
+	display_name = "psionic receiver"
```
Comment (1Sparky1) [inline code/modules/client/preference_setup/loadout/loadout_augments.dm:74]:
```suggestion
	display_name = "memory inhibitor"
```
wrong name here.

**Review by 1Sparky1 — CHANGES_REQUESTED** [review]

**Review by Geevies — APPROVED** [review]

**Comment by niennab** [comment]
Just to log it here, I wanted to request IPCs getting the:
Vision Enhanced Retinas, Ocular Installed Sightlights and Zeng-Hu Veterancy Plate (so long as it works in testing)

**Review by Arrow768 — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
Bot looks busted here, so I'm just gonna do the following.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**